sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -2124,6 +2124,9 @@ class Analyzer(
     override def apply(plan: LogicalPlan): LogicalPlan = plan.transformUp {
       case p if p.resolved => p
       case p => p transformExpressionsUp {
+        // Produces a placeholder random seed for streaming query, the real random seed
+        // is given at the beginning of Optimizer.
+        case Uuid(None) if p.isStreaming => Uuid(Some(-1))
```

Comment:
`Uuid` need to have a random seed initialized to make it resolved. This gives it a fake seed. Since we assign random seeds at optimizer, we can get rid of it. The intent here is to have a placeholder seed shown in analyzed plan. Not a big deal, so I'm going to remove it.